src/Runner.Common/ResultsServer.cs

```diff
@@ -52,8 +53,8 @@ public sealed class ResultServer : RunnerService, IResultsServer
 
         public void InitializeResultsClient(Uri uri, string liveConsoleFeedUrl, string token)
         {
-            var httpMessageHandler = HostContext.CreateHttpClientHandler();
-            this._resultsClient = new ResultsHttpClient(uri, httpMessageHandler, token, disposeHandler: true);
+            this._resultsClient = CreateHttpClient(uri, token);
```

Comment:
should we have 2 clients for result server, one with Token, one with the NoOpsCredential?